src/mbgl/tile/geometry_tile_worker.cpp

```diff
@@ -170,6 +168,26 @@ void GeometryTileWorker::setLayers(std::vector<Immutable<LayerProperties>> layer
     }
 }
 
+void GeometryTileWorker::reset(uint64_t correlationID_) {
+    try {
```

Comment:
what exception this might throw?

Reply:
good point, we have quite a few of those in a tile worker.

Reply:
I just took after other methods, no exceptions expected here :) I'll remove try.. catch